### ISSUE TITLE
Accessible view / Open Detected Link iteration

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode#196644]
Testing https://github.com/microsoft/vscode/issues/193988

It seems like the accessible buffer _always_ open in screen reader accessibility mode after the link quickpick is closed, even if it was not open before. This seems like a bug.

Also: it seems like Gitlens uses Ctrl+Shift+G as the start of its chords. With Gitlens installed, it overrides the keybinding, causing the editor to wait for another keypress.

**Comment by meganrogge** [comment]
Can you pls provide the exact repro steps?

**Comment by connor4312** [comment]
For the first case:

1. Run "Open Detected Link" from the command palette
2. Hit escape
3. 🐛 the accessible buffer is opened

For the second issue, try using the Ctrl+Shift+G in the accessible buffer with Gitlens installed (on Windows at least)

**Comment by meganrogge** [comment]
Re the gitlens issue, most keybindings are taken, so I don't think I'll do anything about this.

**Comment by meganrogge** [comment]
In your steps above, if focus was in the accessible view before step 1, this is as designed. 

I cannot repro when I first focus the terminal.

**Comment by meganrogge** [comment]
Also assuming you pressed escape rather than going to a link

**Comment by connor4312** [comment]
>Also assuming you pressed escape rather than going to a link

That's right

> if focus was in the accessible view before step 1, this is as designed.

Before step 1, I can have my focus in an editor or a terminal and the accessible view still opens. I noticed this only happens after I open the accessible view once:

https://github.com/microsoft/vscode/assets/2230985/daa57577-95db-4708-a84c-36a60c983d7d
